ReactAndroid/src/main/java/com/facebook/react/views/art/ARTSurfaceViewShadowNode.java

```diff
@@ -33,6 +35,18 @@
 
   private @Nullable Surface mSurface;
 
+  private boolean mIsBackgroundColorSet;
+  private int mBackgroundColor;
+
+  @ReactProp(name = ViewProps.BACKGROUND_COLOR, customType = "Color")
+  public void setBackgroundColor(Integer color) {
+    mIsBackgroundColorSet = (color != null);
+    if (mIsBackgroundColorSet) {
```

Comment:
I think if we remove the background color (color = null) it will keep the old background color instead of drawing no background. If we use my suggestion to make `mBackgroundColor` nullable we could just assign here and null check in `drawOutput`.